test/e2e/config/e2e_conf.yaml

```diff
@@ -158,6 +158,18 @@ providers:
       new: --metrics-addr=:8080
     files:
     - sourcePath: "../data/shared/v1.5/metadata.yaml"
+- name: metal3
+  type: IPAMProvider
+  versions:
+  - name: "{go://github.com/metal3-io/ip-address-manager@v1.9}"
```

Comment:
Here, IPAM is slightly different from other providers because we want to test the `main` branch rather than version 1.9.  

However, I'm wondering how that would work. We can use `v1.10.99` instead of `{go://github.com/metal3-io/ip-address-manager@v1.9}`, but how will we obtain the YAML resources for the `main` branch? Should we push them somewhere during our image-building process? As far as I know, we only push images to quay.

Reply:
Oh we have them in the overrides folder:
```
 Potential override file searchFile="/home/metal3ci/.config/cluster-api/overrides/ipam-metal3/v1.10.99/metadata.yaml" provider="ipam-metal3" version="v1.10.99"
 Using override="metadata.yaml" provider="ipam-metal3" version="v1.10.99"
```
We used them in dev-env, we can use these ones.